### PR TITLE
[14.0] shopfloor_ui_tests: implement jest unit tests

### PR DESCRIPTION
**Pull request by JuMiSanAr**
Very simple implementation of unit tests for the shopfloor app, using jest.

### CODE REVIEW

**Comment by simahawk** [comment]
Nice! Can you give us an example for Vue.component test?

**Comment by JuMiSanAr** [comment]
> Nice! Can you give us an example for Vue.component test?

Example for component testing added!

The tests are also integrated in github.

File: .github/workflows/ui-unit-tests.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: unit tests
```
Comment (simahawk) [inline .github/workflows/ui-unit-tests.yml]:
```suggestion
name: Shopfloor app unit tests
```

**Review by simahawk — APPROVED** [review]

**Pull request merged**